File: 프로그래머스/0/181908. 접미사인지 확인하기/접미사인지 확인하기.java
class Solution {
    public int solution(String my_string, String is_suffix) {
        int begin = my_string.length() - is_suffix.length();
        if(my_string.length() < is_suffix.length()) {
            return 0;
        } else if(my_string.substring(begin).equals(is_suffix)) {
            return 1;
        }
        return 0;
    }
}

File: 프로그래머스/0/181908. 접미사인지 확인하기/접미사인지 확인하기.js
function solution(my_string, is_suffix) {
    return my_string.endsWith(is_suffix) ? 1 : 0;
    //return +my_string.endsWith(is_suffix);
    //return my_string.slice(my_string.length - is_suffix.length) === is_suffix ? 1 : 0;
    //return my_string.substr(-is_suffix.length) === is_suffix ? 1 : 0;
//     return +Boolean([my_string].find(
//         (e) => my_string.slice(my_string.length - is_suffix.length) === is_suffix));
}